spec/controllers/application_controller/feature_spec.rb

```diff
@@ -1,25 +1,21 @@
-require File.expand_path("../app/controllers/application_controller/", "feature.rb")
-
 describe ApplicationController do
```

Comment:
I think what's happening is that action_controller/test_case.rb is bringing in a Feature class under ApplicationController and we *also* happen to have a Feature class.  If so, that would be really terrible...maybe we should rename our class or just call it MiqFeature :trollface: .  Additionally that might explain that funny require that was at the top of the file.